ncm-sysconfig/src/main/perl/sysconfig.pm

```diff
@@ -1,110 +1,118 @@
 #${PMcomponent}
 
-use parent qw(NCM::Component);
+=head1 NAME
+
+sysconfig: management of sysconfig files
+
+=head1 DESCRIPTION
+
+The I<sysconfig> component manages system configuration files in
+C<< /etc/sysconfig >> . These are files which contain key-value pairs.
+However there is the possibility to add verbatim text either before or after the key-value pair definitions.
+
+=cut
+
+use parent qw(NCM::Component CAF::Path);
 our $EC = LC::Exception::Context->new->will_store_all;
 
-use LC::Check;
+use Readonly;
 
-use File::Path;
-use File::Basename;
+Readonly my $QUOTE => "\"";
+Readonly my $SYSCONFIGDIR => "/etc/sysconfig"; # The base directory for sysconfig files.
 
```

Comment:
@jrha well, almost nice. can you also add 
```
our $NoActionSupported = 1;
```
(as per @ned21 request)

Reply:
Ugh, I had added that already... where did it go?!

Reply:
Done.